DOCSP-46643: JVM future minor release dropping 4.0 (#91)
10gen/docs-shared: dbx/jvm/v5.3-wn-items.rst

- :red:`Important:` The next minor release will drop support for MongoDB
  Server version 4.0 and raise the minimum supported version to 4.2. We

>   recommend making changes to your application if you might be affected.

Reviewer: I think we should keep the statement more concrete. I also switched the affected thing to the application in this suggestion, but feel free to workshop.

```suggestion
  recommend making changes to your application if it will be affected.
```

Author: reworded
